modules/cli/cmd/cliflaghandler.go

```diff
@@ -133,3 +165,21 @@ func (f *CliFlagHandler) GetValue(flagName string) string {
 	log.Debugf("%s flag is not set, using default: %s", flagName, value)
 	return value
 }
+
+// Helper function to get field by tag
+func getFieldByTag(obj interface{}, tagName, tagValue string) (reflect.Value, bool) {
```

Comment:
```suggestion
func getFieldValueByTag(obj interface{}, tagName, tagValue string) (reflect.Value, bool) {
```
I think this name more accurately reflects what this function is doing.